main/createUiDefinition.json

```diff
@@ -69,6 +69,18 @@
           "validationMessage": "Insert valid Jointoken."
         },
         "visible": "true"
+      },
+      {
+        "name": "accessproxy",
+        "type": "Microsoft.Common.TextBox",
+        "label": "http proxy",
+        "toolTip": "Pass Accessproxy.",
+        "constraints": {
+          "required": false,
+          "regex": "(http|https)?:\\/\\/(\\S+)",
```

Comment:
does this allow `http://xyz.com:8080` ?

Reply:
yes this checks whether address starts from http or https and does not have a whitespace in between